software/M03_betas.py

```diff
@@ -105,7 +105,7 @@ def run(args):
     if args.output_folder or args.output:
         if args.output_folder and not os.path.exists(args.output_folder):
             os.makedirs(args.output_folder)
-        else:
+        elif args.output:
             Utilities.ensure_requisite_folders(args.output)
```

Comment:
```suggestion
            Utilities.ensure_requisite_folders(args.output)
        else:
            raise Exceptions.InvalidArguments("Unsupported output parameters")
```

Reply:
This suggestion is not strictly necessary with the current code, but it makes it more resilient if it gets changed in the future.
Feel free to include it or discard it in this pull request.